quickstarts/05-multi-agent-workflow-dapr-workflows/services/elf/app.py

```diff
@@ -20,8 +20,7 @@ async def main():
             state_key="workflow_state",
             agents_registry_store_name="agentstatestore",
             agents_registry_key="agents_registry",
-            service_port=8003,
-        )
+        ).as_service(port=8003)
```

Comment:
can you not make this app a service, so we can have an example where all apps work using streaming subs? ty